gatsby-browser.ts

```diff
@@ -0,0 +1,7 @@
+import type { GatsbyBrowser } from "gatsby";
+
+export const onRouteUpdate: GatsbyBrowser["onRouteUpdate"] = ({ location }) => {
+  if (window.parent) {
+    window.parent.postMessage(location.pathname);
+  }
+};
```

Comment:
This allows the parent iframe to track Gatsby's page changes for contextual editing.